Code/Tools/ProjectManager/Platform/Linux/ProjectUtils_linux.cpp

```diff
@@ -45,10 +45,42 @@ namespace O3DE::ProjectManager
                     return AZ::Success(supportClangCommand);
                 }
             }
-            return AZ::Failure(QObject::tr("Clang not found. \n\n"
+            return AZ::Failure(QObject::tr("Clang not found. <br><br>"
                 "Make sure that the clang is installed and available from the command prompt. "
                 "Refer to the <a href='https://o3de.org/docs/welcome-guide/setup/requirements/#cmake'>O3DE requirements</a> page for more information."));
         }
+
+
+        AZ::Outcome<void, QString> OpenCMakeGUI(const QString& projectPath)
+        {
+            AZ::Outcome processEnvResult = GetCommandLineProcessEnvironment();
+            if (!processEnvResult.IsSuccess())
+            {
+                return AZ::Failure(processEnvResult.GetError());
+            }
+
+            QString projectBuildPath = QDir(projectPath).filePath(ProjectBuildPathPostfix);
+            AZ::Outcome projectBuildPathResult = GetProjectBuildPath(projectPath);
+            if (projectBuildPathResult.IsSuccess())
+            {
+                projectBuildPath = projectBuildPathResult.GetValue();
+            }
+
+            QProcess process;
+            process.setProcessEnvironment(processEnvResult.GetValue());
+
+            // if the project build path is relative, it should be relative to the project path 
+            process.setWorkingDirectory(projectPath);
+
+            process.setProgram("cmake-gui");
+            process.setArguments({ "-S", projectPath, "-B", projectBuildPath });
```

Comment:
For Linux, this will need the CMAKE_C_COMPILER and CMAKE_CXX_COMPILER set as well. 
```
.. -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12
```

Or even better, the above can be ignored if the following environment values are set
```
CC=clang-12
CXX=clang++-12
```
You can set this in the `GetCommandLineProcessEnvironment` function